docker/puppet-runtime/spec/container_spec.rb

```diff
@@ -1,14 +1,14 @@
 require 'rspec/core'
+require 'pupperware/spec_helper'
+include Pupperware::SpecHelpers
 
 describe 'puppet-agent runtime container' do
-  include Pupperware::SpecHelpers
-
   before(:all) do
     @image = require_test_image
   end
 
   it 'should have ruby' do
     result = run_command("docker run #{@image} /opt/puppetlabs/puppet/bin/ruby --version")
-    expect(result[:status]).to eq(0)
+    expect(result[:status].exitstatus).to equal(0)
```

Comment:
Interesting, I thought `equal` was for identity comparison? But it works for integers of the same value?

https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/equality-matchers#compare-using-equal-(equal?)
https://apidock.com/ruby/Object/equal%3F